aiodocker/docker.py

```diff
@@ -320,6 +320,30 @@ def __init__(self, docker):
         )
         return DockerContainer(self.docker, id=data['Id'])
 
+    async def run(self, config, name=None):
```

Comment:
I like the addition of the high-level `run` API for convenience.
Though, in future revisions, I would move this to a separate collection of high-level interfaces.